packages/cloud-functions/.gitignore

```diff
@@ -0,0 +1 @@
+*/yarn.lock
```

Comment:
@chrismaree now that these are in the yarn workspace, you should be able to drop this I think....? yarn shouldn't produce a lock file I don't think (at least when running in the repo).

Reply:
good point! have removed it.